.github/workflows/node.js.yml

```diff
@@ -10,6 +10,42 @@ on:
         branches: [master, feature/*, mynah-dev]
 
 jobs:
+    check-test-files:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  fetch-depth: 2
+            - name: Check for the presence test files
+              id: check_file
+              run: |
+                  # Check if there are any changes in the src folder
+                  sourceFiles=$(git diff --name-only ${{ github.event.pull_request.base.sha }} | { grep 'src' || true; })
+                  if [ -z "$sourceFiles" ]; then
```

Comment:
Just for reference: if this script gets bigger, we can move it to a .js file and call it from this `actions/github-script` job as described in: https://github.com/actions/github-script?tab=readme-ov-file#run-a-separate-file

Reply:
Just changed to a script, I think it's going to be easier to maintain